package.json

```diff
@@ -7,7 +7,13 @@
     "test:open": "npm run lint && cypress open",
     "test": "cypress run",
     "lint": "eslint ./cypress/integration/**/*.ts",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "delete:reports": "rm cypress/results/* || true",
+    "prereport": "npm run delete:reports",
+    "combine-reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+    "generate-report": "marge cypress/reports/report.json -f report -o cypress/reports",
+    "report": "cypress run --reporter cypress-multi-reporters --reporter-options configFile=reporter-config.json",
+    "postreport": "npm run combine-reports"
```

Comment:
The idea here is:
1. Add the script to delete the reports (done)
2. Add the combine-reports and generate-report scripts (done)
3. Use the pre and post hooks for the test command, so we don't need a report script. In the pre hook script, you should call the delete script, and in the post hook, you should call the combine and generate reports scripts.